actors/builtin/paych/paych_actor.go

```diff
@@ -210,11 +211,13 @@ func (pca Actor) UpdateChannelState(rt vmr.Runtime, params *UpdateChannelStatePa
 				Nonce:    0,
 			}
 			st.LaneStates = append(st.LaneStates[:laneIdx], append([]*LaneState{ls}, st.LaneStates[laneIdx:]...)...)
-
+			lanFound = false
 		}
 
-		if ls.Nonce > sv.Nonce {
-			rt.Abortf(exitcode.ErrIllegalArgument, "voucher has an outdated nonce, cannot redeem")
+		if lanFound {
+			if ls.Nonce >= sv.Nonce {
+				rt.Abortf(exitcode.ErrIllegalArgument, "voucher has an outdated nonce, cannot redeem")
```

Comment:
Nit: include the relevant values in the message

Reply:
This is done.